test/utils.jl

```diff
@@ -1,7 +1,7 @@
 @testset "utils" begin
     using KernelFunctions: vec_of_vecs, ColVecs, RowVecs
     rng, N, D = MersenneTwister(123456), 10, 4
-    x, X = randn(rng, N), randn(rng, D, N)
+    x, X, v, w = randn(rng, N), randn(rng, D, N), randn(rng, D), randn(rng, N)
```

Comment:
```suggestion
    x = randn(rng, N)
    X = randn(rng, D, N)
    v = randn(rng, D)
    w = randn(rng, N)
```

I just find this easier to comprehend